invenio_communities/templates/invenio_communities/mycommunities.html

```diff
@@ -10,10 +10,13 @@
 <div class="well">
 {% if current_user.is_authenticated %}
   {% if mycommunities %}
+  {% if not can_user_create_community(current_user) %}
+    <p class="alert alert-warning small">To create a community your account must be verified for at least one week. If you want to speed up the process, you can contact our support team describing your case.</p>
+  {% endif %}
   <h2>
     {{ _('My communities') }}
     <div class="pull-right">
-      &nbsp;<a href="{{url_for('.new')}}" class="btn btn-primary"><i class="icon-plus-sign"></i> {{ _('New') }}</a>
+      &nbsp;<a href="{{url_for('.new')}}" class="btn btn-primary" {{ "disabled" if not can_user_create_community(current_user) }}><i class="icon-plus-sign"></i> {{ _('New') }}</a>
```

Comment:
```suggestion
      &nbsp;<a href="{{url_for('.new')}}" class="btn btn-primary {{ "disabled" if not can_user_create_community(current_user) }}" {{ "disabled" if not can_user_create_community(current_user) }}><i class="icon-plus-sign"></i> {{ _('New') }}</a>
```

This also makes the button unclickable